test/suite/test_tiered06.py

```diff
@@ -105,8 +106,7 @@ def test_ss_basic(self):
 
         # The object doesn't exist yet.
         if self.ss_name == 's3_store':
-            with self.expectedStderrPattern('.*HTTP response code: 404.*'):
-                self.assertFalse(fs.fs_exist(session, 'foobar'))
+            self.assertFalse(fs.fs_exist(session, 'foobar'))
```

Comment:
We don't need the if-else here. We can remove lines 108, 110 and 111.

Reply:
Fixed.